Makefile

```diff
@@ -88,7 +88,7 @@ style: style-rust style-python
 
 style-rust:
 	@rustup component add rustfmt --toolchain stable 2> /dev/null
-	cargo +stable fmt -- --check
+	cargo +stable fmt --all -- --check
```

Comment:
should probably use `--workspace` instead of `--all`?

applies to all others uses of `--all` to

Reply:
There is no `--workspace` on `fmt`. Honestly, I don't know what `--all` does on fmt, but it takes longer and we also use the same in `sentry-rust`.